rest-api/tools/import_organizations.py

```diff
@@ -121,16 +138,91 @@ def _entity_from_row(self, row):
 class SiteImporter(CsvImporter):
 
   def __init__(self):
-    super(SiteImporter, self).__init__('site', SiteDao(), 'siteId', 'googleGroup',
-                                       [SITE_ORGANIZATION_ID_COLUMN, SITE_SITE_ID_COLUMN,
-                                       SITE_SITE_COLUMN, SITE_STATUS_COLUMN,
-                                       ENROLLING_STATUS_COLUMN, DIGITAL_SCHEDULING_STATUS])
-
-    self.organization_dao = OrganizationDao()
     args = parser.parse_args()
+    self.organization_dao = OrganizationDao()
     self.geocode_flag = args.geocode_flag
     self.ACTIVE = SiteStatus.ACTIVE
     self.status_exception_list = ['hpo-site-walgreensphoenix']
+    self.project = args.project
+    self.instance = args.instance
+    self.creds_file = args.creds_file
+    self.new_sites_list = []
+
+    if self.project in ENV_LIST:
+      self.environment = ' ' + self.project.split('-')[-1].upper()
+    elif self.project == ENV_LOCAL:
+      self.environment = ' ' + ENV_STABLE.split('-')[-1].upper()
```

Comment:
FYI, this logic is going to get invoked when circle runs import_organizations.sh as a part of its normal test.sh logic on the circle instance. So that's why you're hitting the import error.

Reply:
removed localhost references